presto-spi/src/main/java/com/facebook/presto/spi/resourceGroups/ResourceGroupConfigurationManager.java

```diff
@@ -15,16 +15,16 @@
 
 import java.util.Optional;
 
-public interface ResourceGroupConfigurationManager
+public interface ResourceGroupConfigurationManager<C>
```

Comment:
Add Javadoc explaining the flow: the engine calls match(), then creates a resource group and calls configure() if the resource group ID in the selector context does not exist.